test/parallel/test-path-resolve.js

```diff
@@ -9,6 +9,14 @@ const failures = [];
 const slashRE = /\//g;
 const backslashRE = /\\/g;
 
+const posixyCwd = common.isWindows ?
+  (() => {
+    const _ = process.cwd()
+      .replaceAll(path.sep, path.posix.sep);
+    return _.substr(_.indexOf(path.posix.sep));
```

Comment:
Nit: prefer slice. Substr is deprecated/legacy

Reply:
Sure, moved to slice().